tests/filesTest.js

```diff
@@ -1122,20 +1237,19 @@ describe('Main: Currently testing files management,', function () {
   describe('unfavorite a file', function () {
     it('checking method: unfavorite', async function () {
       const provider = createProvider(true, true)
```

Comment:
should pass now in oc10, can you set it to `const provider = createProvider(false, true)`